.github/workflows/test.yml

```diff
@@ -94,8 +96,8 @@ jobs:
         shell: cmd
         if: matrix.os == 'windows-latest'
 
-  test_ros_binary_install_ubuntu:
-    name: "Check ROS distribution"
+  test_ros_binary_install_linux:
+    name: "Check ROS distribution '${{ matrix.ros_distribution }}'' on '${{ matrix.docker_image }}'"
```

Comment:
You don't really need to mention the values in the job name; they're included in parentheses, e.g., "Test setup-ros / Check ROS distribution (windows-latest, jazzy)"

Reply:
That did not work. E.g. in https://github.com/ros-tooling/setup-ros/actions/runs/9487986336 the name only mentions the ROS distribution and not the Linux distribution. In effect, it did not test the ROS distribution on AlmaLinux.

Reply:
Try adding the key + list of possible values directly under `matrix:`, like we have for `ros_distribution:`.

Reply:
E.g.:

```yaml
      matrix:
        ros_distribution:
          - noetic
          - humble
          - iron
          - jazzy
          - rolling
        docker_image:
          - ubuntu:focal
          - ubuntu:jammy
          - ubuntu:noble
          - almalinux:8
          - almalinux:9
```

Reply:
Yes, that would work. I decided against this because this then requires a constant sync between this list and the values in the `include`.

Reply:
As you wish.

Reply:
Wait. Doesn't it look like some of the "Test setup-ros / Check ROS distribution ..." jobs are missing? There's only one job per distro. There should be one `ubuntu:noble` job for `rolling` and one `almalinux:9` job with `rolling`, but I only see `almalinux:9`. Same for `humble`. I think you might have to add the `docker_image:` list after all :/

Reply:
Argh. You are right. Almost all CI jobs start with "Check". It's hard to spot this issue. I added the `docker_image` list. But I think there should be a better way to test Linux - ROS pairs on the CI. Now it also tries to build pairs that are not in the `include`. And adding now extra `exclude` is a maintenance nightmare.

Reply:
And now we have jobs with combinations we don't want. What if you keep `docker_image:` but remove its elements?

Reply:
Yeah. I believe you might just need to mention the keys under `matrix:` and then you can just use `include:`?

Reply:
or removing everything under `matrix:` works I guess :laughing:

Reply:
> or removing everything under `matrix:` works I guess 😆

Yes. Removing things sometimes works better than adding things.